src/primitives/transaction.h

```diff
@@ -190,7 +190,8 @@ class CTxOut
             nSize += (32 + 4 + 1 + 107 + 4); // the 148 mentioned above
         }
 
-        return 3 * minRelayTxFee.GetFee(nSize);
+        // Dogecoin: Anything below 1 DOGE is always dust
+        return std::max(COIN, 3 * minRelayTxFee.GetFee(nSize));
```

Comment:
minRelayTxFee as passed from GetDogecoinDustFee() is 1 COIN, so this would return a threshold of 3*COIN 🤔

Reply:
Okay so setting this to always return COIN will indeed roll a half-COIN change into fee. This was tested with a single input tx on regtest.

Reply:
Okay, going to have it just do that. Also; it looks like we may have a nightmare to unpick all this to enable Segwit